src/include/verbose.h

```diff
@@ -64,11 +79,38 @@ typedef enum {
     WT_VERBOSE_DEBUG
 } WT_VERBOSE_LEVEL;
 
+/* Convert a verbose level to its string representation. */
+#define WT_VERBOSE_LEVEL_STR(level, level_str) \
+    do {                                       \
+        level_str = "";                        \
```

Comment:
Since you're doing it elsewhere, parenthesize macro arg `level_str`

Reply:
I should indeed! Fixed in 16c7aca2f41c8d211059fa909d2ed4dad669f623